src/main/java/parsers/Parser.java

```diff
@@ -0,0 +1,32 @@
+package parsers;
+
+public class Parser {
+
+    /*
+     * Returns a String with the specific command.
+     *
+     * @params userInput The input from the user containing the command.
+     * @return A String containing just the command.
+     */
+    public String parseCommand(String userInput) {
```

Comment:
Would you consider removing the whitespaces around the input just in case it is accidentally keyed in